sycl/doc/developer/DockerBKMs.md

```diff
@@ -41,17 +41,20 @@ The following containers are publicly available for DPC++ compiler development:
 - `ghcr.io/intel/llvm/ubuntu2404_base`: contains basic Ubuntu 24.04 environment
    setup for building DPC++ compiler from source.
 - `ghcr.io/intel/llvm/ubuntu2204_intel_drivers`: contains everything from the
-   base container + pre-installed Intel drivers.
-   The image comes in four flavors/tags:
+   Ubuntu 22.04 base container + pre-installed Intel drivers.
+   The image comes in two flavors/tags:
    * `latest`: Intel drivers are downloaded from release/tag and saved in
     dependencies.json. The drivers are tested/validated everytime we upgrade
     the driver.
+   * `alldeps`: Includes the same Intel drivers as `latest`, as well as the
+   development kits for NVidia/AMD from the `ubuntu2204_build` container.
+- `ghcr.io/intel/llvm/ubuntu2404_intel_drivers`: contains everything from the
+   Ubuntu 24.04 base container + pre-installed Intel drivers.
+   The image comes in two flavors/tags:
```

Comment:
I believe we should provide `latest` and `alldeps` for Ubuntu 24.04 as well. Any plans for enabling them in the future?

Reply:
sure, ill do it after i fix the igc-dev builds which is my overall goal with these PRs